main.go

```diff
@@ -398,30 +390,33 @@ func usageMessage() {
 /* listAll = true - all versions including beta and rc will be displayed */
 /* listAll = false - only official stable release are displayed */
 func installOption(listAll bool, custBinPath, mirrorURL *string) {
-	tflist, _ := lib.GetTFList(*mirrorURL, listAll) //get list of versions
-	recentVersions, _ := lib.GetRecentVersions()    //get recent versions from RECENT file
-	tflist = append(recentVersions, tflist...)      //append recent versions to the top of the list
-	tflist = lib.RemoveDuplicateVersions(tflist)    //remove duplicate version
+	tflist, _ := lib.GetTFReleases(*mirrorURL, listAll) //get list of versions
+	recentVersions, _ := lib.GetRecentVersions()        //get recent versions from RECENT file
+	var versions []string
+	for _, r := range tflist {
+		versions = append(versions, r.Version)
+	}
+	versions = append(recentVersions, versions...)
+	versions = lib.RemoveDuplicateVersions(versions) //remove duplicate version
 
 	if len(tflist) == 0 {
-		fmt.Println("[ERROR] : List is empty")
-		os.Exit(1)
+		log.Fatalln("[ERROR] : List is empty")
 	}
 	/* prompt user to select version of terraform */
 	prompt := promptui.Select{
 		Label: "Select Terraform version",
-		Items: tflist,
+		Items: versions,
 	}
```

Comment:
Is there a way in Go to prompt the version from within `tflist` rather than creating a separate slice here?